### ISSUE TITLE
[FEATURE] Update download page for consistency across the website

### ISSUE THREAD

**Issue by FabrizioJordan** [SaptarshiSarkar12/Drifty#734]
### Is your feature request related to a problem? Please describe.

The download page looks old and does not align with a modern design and style.
And the page uses bad practices such as buttons with gradient backgrounds that cannot have animations or transitions, and the buttons have animations with poor accessibility.

### Describe the solution you'd like.

Everything with better separation:
I replaced the incorrectly used padding and margin with flex and gap.

This changes are for the Download Page (Realases.js):
I fixed the bad practices in the buttons:
The background now is with one color and in the hover dont have a great change. (Nomally bg-[#1559f0] , in hover: bg-[#0e2ba3])
This change is because simplicity and clarity are better than gradients, and gradient colors cannot be animated in pure CSS.
No scale and no translate animations, because this is not good in a modern web.

I've fixed a bug in the  latest version buttons that could overflow the page.

Buttons are now more accessible and clearer.

In the rest of Download page:
The background of the page now is more simple and clear.

All of the colors are subject to change (i use this colors for a example).

A new "hr" to separate the latest and older versions.

More:
Minor changes in Main Section.

### Download page:

Mobile:
![image](https://github.com/user-attachments/assets/7dc0b839-d876-467e-8d49-511ca4ecc649)

Tablet:
![image](https://github.com/user-attachments/assets/91060780-3c8f-4dbf-8246-73b6c321ef08)

PC:
![image](https://github.com/user-attachments/assets/f2f38535-2062-4445-aaf0-826aee19ed7c)

Latest Releases:
![image](https://github.com/user-attachments/assets/6ecebc5a-0acb-4740-92d0-c1594c45b08b)

Past Releases:
![image](https://github.com/user-attachments/assets/0fa5d564-e358-4ea2-b4f9-4bc9adb5b1e0)

Latest Releases with one button normal, one on hover and one on active state:
![image](https://github.com/user-attachments/assets/7203677c-90ce-4c28-acf4-92300d01e064)

### Additional information

All colors are just to demonstrate a change, they may not be the best colors.

### Do you want to work on this issue?

Yes

**Comment by SaptarshiSarkar12** [comment]
@FabrizioJordan Hi Fabricio :wave:!
I appreciate the changes especially on the accessibility side. But I recommend choosing any other colours for the buttons. Also, I think it would be better to distinguish between the colours of the buttons in the latest release and that of the past releases, wouldn't it?

**Comment by FabrizioJordan** [comment]
Sorry for the delay, I had some things to finish, and I was just completing them.

Are the colors like this okay?
If you want a specific color, let me know, and we can discuss if it works or if it might cause accessibility issues.

![image](https://github.com/user-attachments/assets/e461d553-aeb2-45b0-b22d-49bb5ba7a211)

![image](https://github.com/user-attachments/assets/0ac49f53-d275-4b6a-96e4-20d9063364b2)


I’m not sure if you prefer a light color or a dark color for these buttons in the end.


I think the colors of these buttons are not very important - they are significantly far apart, and the 'latest releases' buttons are much bigger.
But this is a good change to make. Let me know what you think. (I wouldn’t know how to distinguish one from the other beyond this without making them look visually unappealing and breaking the palette)

**Comment by SaptarshiSarkar12** [comment]
> Sorry for the delay, I had some things to finish, and I was just completing them.

It's fine :smile:.

> Are the colors like this okay? If you want a specific color, let me know, and we can discuss if it works or if it might cause accessibility issues.

@FabrizioJordan Yeah, anything that can look appealing. I think the colour of the button for latest release does not have a good contrast with the background, does it?

> I’m not sure if you prefer a light color or a dark color for these buttons in the end.

I do prefer dark as most devs do :upside_down_face:. Can you show me an example for dark colour? I can compare.
 
> I think the colors of these buttons are not very important - they are significantly far apart, and the 'latest releases' buttons are much bigger. But this is a good change to make. Let me know what you think. (I wouldn’t know how to distinguish one from the other beyond this without making them look visually unappealing and breaking the palette)

You are right :+1:. No change needed then :)

**Comment by FabrizioJordan** [comment]
> I think the colour of the button for latest release does not have a good contrast with the background, does it?

That's right.

> I do prefer dark as most devs do 🙃. Can you show me an example for dark colour? I can compare.

I’m not sure what color to use for these buttons, but here are some examples I came up with.

1. ( #151e8f )
![image](https://github.com/user-attachments/assets/9534b140-de87-40bd-b715-2687daf5375e)

2. ( #1d29bc )
![image](https://github.com/user-attachments/assets/be6ce6df-587d-4598-b2fc-e87c7e6a91c3)

3. ( #0313d5 )
![image](https://github.com/user-attachments/assets/2431d371-178c-4146-8a06-99176bc151f3)

4. ( #2c59bc )
![image](https://github.com/user-attachments/assets/0571729d-854c-4915-85bb-c0e0ccede36f)

I’m not sure if you know how to preview these directly on the website to decide which one works best, but feel free to adjust or change these colors.

**Comment by SaptarshiSarkar12** [comment]
@FabrizioJordan I apologise for the late reply. I was very busy in the last weekend.
> I’m not sure if you know how to preview these directly on the website to decide which one works best, but feel free to adjust or change these colors.

I tried like this to preview the colours. Is this what you are talking about?
![Screenshot From 2024-12-13 13-35-06](https://github.com/user-attachments/assets/552c34e0-538a-406f-a731-3f4b2791c660)

So, I think we must use the 4th colour (#2c59bc) as the button colour and the 1st one as the colour for on-hover effect. What do you think? Let me know your choice of colours for this scenario.

**Comment by FabrizioJordan** [comment]
> I tried like this to preview the colours. Is this what you are talking about?

Yes, this devtools are great for previewing potential changes to the website

> So, I think we must use the 4th colour (#2c59bc) as the button colour and the 1st one as the colour for on-hover effect. What do you think? Let me know your choice of colours for this scenario.

I’m not a big fan of the 4th color for the button, but it’s okay. I’m still unsure about the best colors for me.

Here’s how it looks if I replace the colors with the 4th as the normal button color and the 1st as the hover effect color:

![image](https://github.com/user-attachments/assets/df50a1b8-7f8f-40c7-b93d-f05949f6cb1c)

Left in hover, and right normal.


Regardless of which one you want, just let me know and I’ll make it. If the way it looks in the picture is what you want, then I think the PR is ready.

**Comment by FabrizioJordan** [comment]
Hi @SaptarshiSarkar12 ! 
Sorry for the delay in sending the PR, but I’ve had very little free time.

I hope to submit the PR within the next few hours and continue helping with the website as long as my schedule allows.

**Comment by SaptarshiSarkar12** [comment]
@FabrizioJordan It's okay 😅. Remember to pull down the "master" branch changes.
